rfd/0111-user-cert-accepts-mfa-authn-response.md

```diff
@@ -0,0 +1,33 @@
+---
+authors: Lisa Kim (lisa@goteleport.com)
+state: draft
+---
+
+# RFD 0111 - GenerateUserCerts accepts and validates a MFA authentication response
```

Comment:
This title seems more like an implementation detail than a description of the proposed change.

How about something like:

```suggestion
# RFD 0111 - Support connection testers when per-session MFA is enabled
```

Reply:
Same goes for the PR title :-)

Reply:
ah right thanks!